opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/aggregate.py

```diff
@@ -123,20 +139,25 @@ def __init__(self):
         self.mmsc = MinMaxSumCountAggregator()
         self.current = None
         self.checkpoint = self._TYPE(None, None, None, 0, None)
+        self.last_update_timestamp = None
 
     def update(self, value):
         self.mmsc.update(value)
         self.current = value
+        self.last_update_timestamp = time_ns()
 
     def take_checkpoint(self):
         self.mmsc.take_checkpoint()
         self.checkpoint = self._TYPE(*(self.mmsc.checkpoint + (self.current,)))
 
     def merge(self, other):
         self.mmsc.merge(other.mmsc)
-        self.checkpoint = self._TYPE(
-            *(
-                self.mmsc.checkpoint
-                + (other.checkpoint.last or self.checkpoint.last,)
-            )
-        )
+        last = self.checkpoint.last
+        if self.last_update_timestamp is None:
```

Comment:
it's a complex condition, but I think considering the body of each condition is the same, there maybe a simpler way to perform this check?

`if self.last_update_timestamp is None or self.last_update_timestamp < other.last_update_timestamp`

Reply:
ah, I see it's because of the NoneType. One thing you could try is to use 0 or -1 as the None value. Then the comparisons become really easy.

But it looks like these are publicly exposed methods, so maybe not a good idea if the expectation is someone is using it for their logic and expects semantically correct types.

Reply:
Right, I'm not sure if we want to use this sort of custom encoding,.